content/en/monitors/guide/on_missing_data.md

```diff
@@ -0,0 +1,99 @@
+|-----------------------------------------|----------------------------------------------------------------------------------------------------|
+| `"on_missing_data": "show_and_notify_no_data"` | If data is missing Show NO DATA and notify<br>(Formerly, "Notify if data is missing")                       |
+| `"on_missing_data": "show_no_data"`     | If data is missing Show NO DATA<br>(Formerly, "Do not notify if data is missing")                           |
+| `"on_missing_data": "resolve"`          | If data is missing Show OK                                                                       |
+| `"on_missing_data": "default"` if using sum or count aggregation | If data is missing Evaluate as 0 (or other default value)                                  |
+| `"on_missing_data": "default"` if using all other aggregation types | If data is missing Show last known status |
+
+For all the available fields, see the [API Documentation][1].
+
+Here's an example of before and after of a JSON monitor with those fields:
+
+**Before**  
+```json
+{  
+  "name": "CPU usage is high for host $host.value",  
+    "type": "query alert",  
+    "query": "avg(last_5m):100 - avg:system.cpu.idle{$host} > 90",  
+    "message": "A high CPU usage has been detected for host $host.value, which can impact the system performance.",  
+    "tags": [],  
+    "options": {  
+        "thresholds": { "critical": 90 },  
+        "notify_audit": false,  
+        "include_tags": false,  
+        "notify_no_data": true,  
+        "no_data_timeframe": 10  
+    }  
+}
+```
+
+**After**  
+{{< highlight yaml "hl_lines=11" >}}{
+  "name": "CPU usage is high for host $host.value",  
+    "type": "query alert",  
+    "query": "avg(last_5m):100 - avg:system.cpu.idle{$host} > 90",  
+    "message": "A high CPU usage has been detected for host $host.value, which can impact the system performance.",  
+    "tags": [],  
+    "options": {  
+       "thresholds": { "critical": 90 },  
+       "notify_audit": false,  
+       "include_tags": false,  
+       "on_missing_data": "show_and_notify_no_data"  
+    }  
+}  
+{{< /highlight >}}
+
+## SLO monitors
+
+SLOs treat uptime and downtime according to this mapping:
+
+| On Missing Data Configuration | Monitor Status                 | SLO Treatment               |
+|-------------------------------|--------------------------------|-----------------------------|
+| Show OK                       | OK                             | Uptime                      |
+| Show No Data                  | No Data                        | Uptime                      |
+| Show No Data and Notify       | No Data                        | Downtime                    |
+| Show last known status        | Whatever the last status was   | If OK, Uptime<br>If Alert, Downtime |
+| Evaluate as zero              | Depends on the threshold configuration | If OK, Uptime<br>If Alert, Downtime |
```

Comment:
@roxanne-moslehi any additional thoughts to add?

Reply:
Resolving for now